From a17e8d39cf01e26955574acd236b4ef5b54b8c3b Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Mon, 5 Sep 2016 12:34:20 +0200
Subject: [PATCH 2/2] xbmc_pvr_types.h: Fix compilation with gcc6

Backport of upstream commit from master branch:
https://github.com/xbmc/xbmc/commit/c8a346109926fa03bf6882c0b6e3885af0e0f8ed

This patch can be removed when Kodi is bumped to 17.0-Krypton.

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 xbmc/addons/include/xbmc_pvr_types.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xbmc/addons/include/xbmc_pvr_types.h b/xbmc/addons/include/xbmc_pvr_types.h
index c5ab20b..e518de1 100644
--- a/xbmc/addons/include/xbmc_pvr_types.h
+++ b/xbmc/addons/include/xbmc_pvr_types.h
@@ -34,6 +34,7 @@
 #endif
 #include <string.h>
 #include <stdint.h>
+#include <stdio.h>
 
 #include "xbmc_addon_types.h"
 #include "xbmc_epg_types.h"
-- 
2.9.3

